test/applied-filters/applied-core-filters.test.js

```diff
@@ -57,6 +57,8 @@ describe('d2l-labs-applied-core-filters', () => {
 			const elem = await fixture(basic);
 			filter = elem.querySelector('d2l-filter');
 			appliedCoreFilters = elem.querySelector('d2l-labs-applied-core-filters');
+			await filter.updateComplete;
+			await appliedCoreFilters.updateComplete;
```

Comment:
These are now deferred a bit while the localization loads -- if they were the root of the fixture it would automatically await them. I may need to add more of these if tests fail in Sauce.